CSC631-MultiplayerGameDev-Group-Project/TeamFourProject
# [feat-ui-start-screen-first] Make start screen scene load first on executable application

# About
This makes the scene `StartMenuScene` load first on the executable built application.

## Testing this PR
1. Build the application
2. Run the application
3. Confirm that the first scene loaded is the `StartMenuScene` which says 'Press to Play'
